android/src/main/java/com/mrousavy/camera/CameraViewModule.kt

```diff
@@ -227,14 +229,23 @@ class CameraViewModule(reactContext: ReactApplicationContext) : ReactContextBase
           }
           map.putDouble("neutralZoom", 1.0)
 
+          val supportedVideoResolutions: List<Size>
+          val cameraInfos = cameraSelector.filter(cameraProvider.availableCameraInfos)
+          if (cameraInfos.size > 0) {
+            supportedVideoResolutions = QualitySelector
+              .getSupportedQualities(cameraInfos[0])
+              .map { QualitySelector.getResolution(cameraInfos[0], it)!! }
```

Comment:
Since `getSupportedQualities` returns an array sorted from highest to lowest qualities, `supportedVideoResolutions` will end up sorted from highest supported resolution to lowest.

Reply:
Is it the correct order though? As in the same order with the `outputFormats` below?

Reply:
I'm not sure that the ordering of `outputFormats` matters.  For each `size` in an output format (where I believe `size` means photo size), that format will be given a `videoHeight` and `videoWidth` according to the highest video resolution supported on that camera that is less than `size`.